apps/web/app/api/escrow/dispute/assign/route.ts

```
@@ -0,0 +1,240 @@
+			.eq('id', assignedById)
+			.single()
+
+		if (assignerError || !assigner) {
+			return NextResponse.json(
+				{ error: 'Assigner not found' },
+				{ status: 404 },
+			)
+		}
+
+		// Check if the assigner has the required role (admin or dispute_manager)
+		const hasRequiredRole = assigner.user_roles.some((userRole: any) =>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)